react/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:20.10.0-alpine3.18 as node
+
+RUN mkdir /www
```

Comment:
I would move lines 3-8 down to below line 10 to avoid constantly rebuilding the node package layers.